tempo_project/README.md

```diff
@@ -0,0 +1,450 @@
+| `state_level_simplified` | pandas  | 1               | Yes                          | Yes                                            | Not Necessary                        |
+| `simple_profiles`        | pandas  | 1               | Yes                          | Yes                                            | N/A                                  |
+| `annual_summary_conus`   | pandas  | 1               | Yes                          | Yes                                            | N/A                                  |
+| `annual_summary_state`   | pandas  | 1               | Yes                          | Yes                                            | N/A                                  |
+| `annual_summary_county`  | pandas  | 1               | Yes                          | Yes                                            | N/A                                  |
+| `full_dataset`           | PySpark | 1               | Yes                          | Yes                                            | Not Necessary                        |
+| `full_state_level`       | PySpark | 1               | Yes                          | Yes                                            | Not Necessary                        |
+| `state_level_simplified` | PySpark | 1               | Yes                          | Yes                                            | Not Necessary                        |
+| `simple_profiles`        | PySpark | 1               | Yes                          | Yes                                            | N/A                                  |
+| `annual_summary_conus`   | PySpark | 1               | Yes                          | Yes                                            | N/A                                  |
+| `annual_summary_state`   | PySpark | 1               | Yes                          | Yes                                            | N/A                                  |
+| `annual_summary_county`  | PySpark | 1               | Yes                          | Yes                                            | N/A                                  |
+
+#### examples-duckdb.ipynb
+
+Dependencies:
+- python>=3.10
+- jupyter
+- duckdb>=0.9.2
+- pandas
+- plotly
+
+Advantages: DuckDB makes the most efficient use of available resources and is easy to set up.
+
+Limitations: DuckDB is limited to one node and can run out of resources. What datasets you can analyze and what queries you can perform thus depends on the hardware you use.
+
+##### Getting Started
+
+DuckDB easily loads parquet files, including partitioned files assuming the duckdb version is sufficiently recent. For example, a dataset can be loaded and then previewed with the code:
+```python
+import duckdb
+
+def load_table(filepath, tablename):
+    duckdb.sql(f"""CREATE TABLE {tablename} AS SELECT * 
+                     FROM read_parquet('{filepath}/**/*.parquet', hive_partitioning=true, hive_types_autocast=false);""")
```

Comment:
You have semicolons in most of the duckdb examples, here and in the notebook. They aren't required - and aren't present in duckdb documentation (that I've looked at). Would you mind removing them?